### DISCUSSION TITLE
Wierd unicode chars in rendered html?

### DISCUSSION THREAD

**Discussion by Philip2809** [resend/react-email#1418] [General]
Hey!

I get some unicode chars in my rendered html when using the render or renderAsync functions:
![image](https://github.com/resend/react-email/assets/51290462/a0344340-2d16-4f07-ba15-dc8da6f1b10a)

**Comment by gabrielmfern** [comment]
This is probably the characters that the `Preview` component adds in, this is expected, as those characters are meant so that only the content you want goes into the Preview.

Here's the piece of code responsible for this:

https://github.com/resend/react-email/blob/2c2b07034a70fd3e093c8ab0a32e9d2de02f91cd/packages/preview/src/preview.tsx#L35-L41
